src/components/Game.js

```diff
@@ -13,6 +16,23 @@ const Game = () => {
     }
   }).join(' ');
 
+  const addSentence = (formFields) => {
+    const newSentences = [...sentences]
+    newSentences.push(`The ${formFields.adj1} ${formFields.noun1} ${formFields.adverb} ${formFields.verb} the ${formFields.adj2} ${formFields.noun2}`)
```

Comment:
The `adverb` here is not getting pushed into the sentence. 

```suggestion
    newSentences.push(`The ${formFields.adj1} ${formFields.noun1} ${formFields.adv} ${formFields.verb} the ${formFields.adj2} ${formFields.noun2}`)
```